scripts/deploy-schema.sh

```diff
@@ -6,7 +6,7 @@ version=$(scripts/version.sh vega-lite)
 
 pushd ../schema/vega-lite/
 
-git checkout master
```

Comment:
I'm undoing this branch renaming since it looks like it's a script meant to run in a *different* repository

https://github.com/vega/schema

Reply:
Good catch